modules/ztp-things-to-consider-when-creating-custom-resource-policies.adoc

```diff
@@ -0,0 +1,12 @@
+Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="ztp-things-to-consider-when-creating-custom-resource-policies_{context}"]
+= Things to consider when creating custom resource policies
+
+* The custom resources used to create the ACM policies should be defined with consideration of possible overlay to its metadata and spec/data. For example, if the custom resource `metadata.name` does not change between clusters then you should set the `metadata.name` value in the custom resource file. If the custom resource will have multiple instances in the same cluster, then the custom resource `metadata.name` must be defined in the policy template file.
+
+* In order to apply the node selector for a specific machine config pool, you have to set the node selector value as `$mcp` in order to let the policy generator overlay the `$mcp` value with the defined mcp in the policy template.
+
+* Subscription source files will not be changed.
```

Comment:
s/ will not be/ do(es) not -->avoid future tense

Reply:
Changed: "Subscription source files do not change."